src/wp-admin/options-general.php

```diff
@@ -66,9 +66,17 @@
 <td><input name="blogname" type="text" id="blogname" value="<?php form_option( 'blogname' ); ?>" class="regular-text" /></td>
 </tr>
 
+<?php
+/* translators: Site tagline. */
+$sample_tagline = __( 'Just another WordPress site' );
+if ( is_multisite() ) {
+	/* translators: %s: Network title. */
+	$sample_tagline = sprintf( __( 'Just another %s site' ), get_network()->site_name );
+}
```

Comment:
```suggestion
$sample_tagline = sprintf(
	/* translators: %s: WordPress on single site install, Network title for multisite install. */
	__( 'Just another %s site' ),
	is_multisite() ? get_network()->site_name : __( 'WordPress' )
);
```

Maybe not on this ticket but is it worth doing this to reduce the string count by one here?

Actual questions, not a passive agressive change request -- you have far more idea about translating WP than I.

Reply:
Fair question. However, more than 70 translations of "Just another %s site" would not match the current translation of "Just another WordPress site" by inserting the "WordPress" translation in the placeholder.

| Language | Just another WordPress site | Just another %s site | WordPress |
| --- | --- | --- | --- |
| Albanian | Edhe një tjetër sajt WordPress | Thjesht një tjetër sajt %s | WordPress |
| Amharic | ሌላ የወርድፕረስ ጣቢያ ብቻ | ሌላ %s ጣቢያ | ዎርድፕረስ |
| Arabic | موقع ووردبريس عربي آخر | موقع آخر في %s | ووردبريس |
| Aragonese | Unatro puesto realizau con WordPress | Unatro puesto mas de %s | WordPress |
| Armenian | Եւս մեկ WordPress-ի կայք | Եւս մեկ կայք %s ցանցում | WordPress |
| Azerbaijani | Yeni WordPress Saytı | Başqa bir %s saytı | WordPress |
| Bengali (Bangladesh) | আরও একটি ওয়ার্ডপ্রেস সাইট | আরও একটি %s সাইট | ওয়ার্ডপ্রেস |
| Catalan | Un altre lloc gestionat amb el WordPress | Un altre lloc %s  | WordPress |
| Chinese (Hong Kong) | 全新的香港中文版 WordPress 網站 | 全新的 香港中文版 WordPress 網站 %s | WordPress |
| Chinese (Taiwan) | 全新的繁體中文 WordPress 網站 | 全新的繁體中文 WordPress 網站《%s》 | WordPress |
| Czech | Další web používající WordPress | Další nový web v rámci sítě %s | WordPress |
| Danish | Endnu en WordPress-blog | Endnu en %s-side | WordPress |
| Dutch | En nog een WordPress site | Een andere %s site | WordPress |
| Dutch (Belgium) | En nog een WordPress website | Een andere %s website | WordPress |
| Dutch (Formal) | En nog een WordPress site | Een andere %s site | WordPress |
| Estonian | Järjekordne WordPressi veebileht | Järjekordne %s veebileht | WordPress |
| Faroese | Bert ein onnur WordPress síða | Ein onnur %s síða | WordPress |
| Finnish | Ihan uusi WordPress-sivusto | Uusi %s -sivusto | WordPress |
| Frisian | Samar in WordPress side | Samar in oare %s side | WordPress |
| Friulian | Dome un altri sît WordPress | Dome un altri sît di %s | WordPress |
| Galician | Outro sitio WordPress máis | Simplemente outro sitio %s máis | WordPress |
| German | Eine weitere WordPress-Website | Eine weitere %s Website | WordPress |
| German (Austria) | Eine weitere WordPress-Website | Eine weitere %s Website | WordPress |
| German (Formal) | Eine weitere WordPress-Website | Eine weitere %s Website | WordPress |
| German (Switzerland) | Eine weitere WordPress-Website | Eine weitere %s Website | WordPress |
| German (Switzerland, Informal) | Eine weitere WordPress-Website | Eine weitere %s Website | WordPress |
| Hazaragi | یک سایت دیگر با وردپرس فارسی | یک سایت %s  دیگر | وردپرس |
| Hebrew | אתר וורדפרס חדש |  חדש - %s | וורדפרס |
| Hindi | बस एक और WordPress साइट | बस एक और %s साइट | वर्डप्रेस |
| Icelandic | Frábær vefur í WordPress | Frábær %s vefur | WordPress |
| Indonesian | Moto situs Anda bisa diletakkan di sini | Hanya %s situs lain | WordPress |
| Italian | Un nuovo sito targato WordPress | Ecco un altro sito %s | WordPress |
| Javanese | Namung salah siji ning Situs Wordpress liyane | Namung salah siji ning %s situs | WordPress |
| Khmer | គ្រាន់&ZeroWidthSpace;តែជា&ZeroWidthSpace;វេបសាយ WordPress មួយ | គ្រាន់&ZeroWidthSpace;តែ&ZeroWidthSpace;ជា&ZeroWidthSpace;វេបសាយ %s មួយ | WordPress |
| Kurdish (Kurmanji) | Malpereke din a WordPressê | Malpereke dîtir bi rêka %s | WordPress |
| Kurdish (Sorani) | تەنها بریتیە لە ماڵپەڕێکی دیکەی وۆردپرێس. | ته&zwnj;نها %s ماڵپه&zwnj;ڕێک | وۆردپرێس |
| Lao | ພຽງ&ZeroWidthSpace;ເວັບ&ZeroWidthSpace;ເວີດ&ZeroWidthSpace;ເພ&ZeroWidthSpace;ສ&ZeroWidthSpace;ເວັບ&ZeroWidthSpace;ໜຶ່ງ | ພຽງ&ZeroWidthSpace;ເວັບ %s ເວັບ&ZeroWidthSpace;ໜຶ່ງ | ເວີດເພດ |
| Latvian | Tikai vēl viena WordPress vietne | Vēl viena %s vietne | WordPress |
| Lower Sorbian | Jano dalšne sedło WordPress | Jano druge sedło %s | WordPress |
| Malayalam | ശരിക്കും മറ്റൊരു വേഡ്പ്രസ്സ് സൈറ്റ്  | ശരിക്കും മറ്റൊരു %s സൈറ്റ് | വേർഡ്പ്രസ്സ് |
| Moroccan Arabic | موقع ووردبريس عربي آخر | موقع آخر في %s | ووردبريس |
| Myanmar (Burmese) | WordPress ဖြင့် ရေးသားသော ဆိုဒ်သစ်တစ်ခု။ | %s&ZeroWidthSpace; &ZeroWidthSpace;ဆိုဒ်&ZeroWidthSpace;တစ်&ZeroWidthSpace;ခု | WordPress |
| Pashto | یوازې يوه بله ورډ پرېس ویبپاڼه | یوازې یوه بله %s  ویبپاڼه | ورډ پرېس |
| Persian | یک سایت دیگر با وردپرس فارسی | یک سایت %s  دیگر | وردپرس |
| Persian (Afghanistan) | یک جولانما دیگر با وردپرس فارسی | یک جولانما %s  دیگر | وردپرس |
| Polish | Kolejna witryna oparta na WordPressie | Kolejna witryna sieci &amp;#8222;%s&amp;#8221; | WordPress |
| Romanian | Doar un alt site web WordPress | Doar un alt site %s | WordPress |
| Russian | Ещё один сайт на WordPress | Ещё один сайт сети «%s» | WordPress |
| Saraiki | صرف ہک ٻئی ورڈپریس سائٹ | صرف ہک ٻئی %s سائٹ | ورڈ پریس |
| Serbian | Само још једно Вордпресово веб место | Само још једно %s веб место | Вордпрес |
| Silesian | Jeszcze jedyn plac ôd WordPressa | Jeszcze jedyn plac &amp;#8222;%s&amp;#8221; | WordPress |
| Sindhi | صرف هڪ وڌيڪ ورڈپریس  سائيٽ | بس ٻي %s سائٽ | ورڊپريس |
| Sinhala | තවත් එක් WordPress අඩවියක් | තවත් එක් %s අඩවියක් | වර්ඩ්ප්&zwj;රෙස් |
| Spanish (Argentina) | Otro sitio realizado con WordPress | Otro sitio más de %s | WordPress |
| Spanish (Colombia) | Otro sitio realizado con WordPress | Otro sitio más de %s | WordPress |
| Spanish (Costa Rica) | Otro sitio realizado con WordPress | Otro sitio más de %s | WordPress |
| Spanish (Dominican Republic) | Otro sitio realizado con WordPress | Otro sitio más de %s | WordPress |
| Spanish (Ecuador) | Otro sitio realizado con WordPress | Otro sitio más de %s | WordPress |
| Spanish (Guatemala) | Otro sitio realizado con WordPress | Otro sitio más de %s | WordPress |
| Spanish (Mexico) | Otro sitio realizado con WordPress | Otro sitio más de %s | WordPress |
| Spanish (Puerto Rico) | Otro sitio realizado con WordPress | Otro sitio más de %s | WordPress |
| Spanish (Spain) | Otro sitio realizado con WordPress | Otro sitio más de %s | WordPress |
| Spanish (Uruguay) | Otro sitio realizado con WordPress | Otro sitio más de %s | WordPress |
| Spanish (Venezuela) | Otro sitio realizado con WordPress | Otro sitio más de %s | WordPress |
| Swahili | Tovuti ingine ya WordPress | Tovuti %s lingine | WordPress |
| Swedish | Ännu en WordPress-webbplats | Ännu en webbplats hos %s | WordPress |
| Swiss German | E witeri WordPress- Site | E witeri %s Site | WordPress |
| Tagalog | Isang WordPress site | Isa pang %s site | WordPress |
| Thai | แค่เว็บเวิร์ดเพรสเว็บหนึ่ง | แค่เว็บ %s เว็บหนึ่ง | เวิร์ดเพรส |
| Tibetan | WordPress སྤྱད་ནས་བཟོས་པའི་བོད་ཡིག་གི་དྲ་ཚིགས་གསར་བ་ཞིག |  %s སྤྱད་ནས་བཟོས་པའི་བོད་ཡིག་གི་དྲ་ཚིགས་གསར་བ་ཞིག | WordPress |
| Uighur | يەنە بىر WordPress تور بېكىتى | يەنە بىر %s توربېتى | WordPress |
| Ukrainian | Просто ще один сайт на WordPress | Просто ще один %s сайт | WordPress |
| Uzbek | Yana bitta WordPress sayt | Yana bitta %s tarmog'idagi sayt | WordPress |
| Vietnamese | Một trang web mới sử dụng WordPress | Một trang web mới của %s | WordPress |